iele.md

```diff
@@ -1445,11 +1446,6 @@ For each `call*` operation, we make a corresponding call to `#call` and a state-
          <mode> EXECMODE </mode>
          <gas> GAVAIL </gas>
          <output> _ => ACCT , .Ints </output>
-         <account>
-           <acctID> ACCT </acctID>
-           <code> _ => CODE </code>
-           ...
-         </account>
 
     rule <k> #exception STATUS ~> #finishCodeDeposit _ _ REG _ NEW:Bool => #popCallStack ~> #popWorldState ~> #popSubstate ~> #if NEW #then STATUS #else #registerDelta(REG, 1) ~> #load REG STATUS #fi ... </k>
```

Comment:
I guess the `CODE` variable is no longer used by `#finishCodeDeposit` (and maybe thus not used by the hing that calls `#finishCodeDeposit`, so maybe we can remove that variable argument from those things?

Also, did you check whether #mkCall needs similar treatment?

Reply:
You are right, I pushed a commit that removes the unused parameter.

About the second question, `#mkCall` should be fine as the code cell of the callee should have  already been initialized at the creation of the callee contract. So, even when the cell is empty in the configuration, `#getCode` (the builtin that queries the blockchain client) should be able to fetch the code and populate the cell. That was not the case with `#mkCreate` before this change, because the contract in question was just being created and the code cell was empty even in the state of the blockchain client. Hence, `#getCode` in that case would return `#emptyCode`.